run-lint.sh

```diff
@@ -36,8 +36,8 @@ function fmtPrintln() {
 }
 
 ## Get the image of go-feature-flag-lint from source
-fmtPrintln "info" "pulling the image of go-feature-flag-lint:${GO_FEATURE_FLAG_LINT_DOCKER_TAG} from source"
-docker pull gofeatureflag/go-feature-flag-lint:${GO_FEATURE_FLAG_LINT_DOCKER_TAG}
+fmtPrintln "info" "pulling the image of go-feature-flag-cli:${GO_FEATURE_FLAG_LINT_DOCKER_TAG} from source"
```

Comment:
maybe consider renaming this tag to GO_FEATURE_FLAG_CLI_DOCKER_TAG?

Reply:
Good catch!
I've tried it and it exit with a wrong number I have to check why.